### PR TITLE
Add Albatross testnet instructions

### PR DESCRIPTION
**Pull request by shanev**
## TODO
- [x] genesis file
- [x] add persistent peer
- [x] test faucet
- [x] test post and upvote
- [x] test joining as a validator after genesis

### CODE REVIEW

File: albatross-1/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+- Go version v1.14.+
+- Stakebird version : [v0.3.0](https://github.com/public-awesome/stakebird/releases/tag/v0.3.0)
+
+### Install Stakebird
+
+```sh
+> mkdir -p $GOPATH/src/github.com/public-awesome
+> cd $GOPATH/src/github.com/public-awesome
+> git clone https://github.com/public-awesome/stakebird && cd stakebird
+> git checkout v0.3.0
```
Comment (JakeHartnell) [inline albatross-1/README.md:14]: We don't have a tagged release for this yet... I assume you know this but still leaving the comment. :smile_cat:

File: albatross-1/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
+# Albatross Testnet Instructions
+
+## Software Requirements
+
+- Go version v1.14.+
+- Stakebird version : [v0.3.0](https://github.com/public-awesome/stakebird/releases/tag/v0.3.0)
+
```
Comment (JakeHartnell) [inline albatross-1/README.md:7]: Technically, `make` and a gcc compiler are also dependencies... though most people will already have these unless they are starting with a fresh server instance.

**Review by JakeHartnell — APPROVED** [review]

File: albatross-1/README.md (hunk 2 above)
Comment (shanev) [reply albatross-1/README.md:7]: Yeah started a separate doc on setting up an Ubuntu node from scratch: https://www.notion.so/publicawesome/Setting-up-a-Stakebird-Validator-on-Ubuntu-56ab9f4eb1b14fd6a7dc63ee0d763093. Will include that here or post it to the blog.

File: albatross-1/README.md (hunk 1 above)
Comment (shanev) [reply albatross-1/README.md:14]: Yeah :) Assuming it'll be tagged before this becomes a real PR.

File: albatross-1/README.md (hunk 2 above)
Comment (shanev) [reply albatross-1/README.md:7]: Actually might just link to the binary, since we're building for all platforms anyway.

**Review by JakeHartnell — APPROVED** [review]

**Review by jhernandezb — APPROVED** [review]
Looks good to me

**Review by JakeHartnell — APPROVED** [review]

**Pull request merged**